liota/dccs/iotcc.py

```diff
@@ -672,3 +684,9 @@ def on_response(msg):
         self.comms.send(json.dumps(self._get_properties(self.next_id(), resource_uuid)))
         on_response(self.recv_msg_queue.get(True, timeout))
         return self.prop_list
+
+    def _validate_input(self, input):
```

Comment:
If it contains the following "^[A-Za-z0-9\s\._-]+$" characters can the command injection happen in python or only in the shell script scenario?

Reply:
To avoid various possibilities of injections, it was suggested to allow only these white-listed set of characters to be consumed from the user code.